src/library/LocalLoopSetPointTracking.py

```diff
@@ -0,0 +1,46 @@
+"""
+## Local Loop Performance Verification - Set Point Tracking
+
+### Description
+
+This verification checks the set point tracking ability of local control loops.
+
+### Verification logic
+
+With a threshold of 5% of abs(set_point) (if the set point is 0, then the threshold is default to be 0.01), if the number of samples of which the error is larger than this threshold is beyond 5% of number of all samples, then this verification fails; Otherwise, it passes.
+
+### Data requirements
+
+- feedback_sensor: feedback sensor reading of the subject to be controlled towards a set point
+- set_point: set point value
+
+"""
+
+from checklib import RuleCheckBase
+
+
+class LocalLoopSetPointTracking(RuleCheckBase):
+    points = ["feedback_sensor", "set_point"]
+
+    def error_below_5percent(self, t):
```

Comment:
Should the percentages be defined through parameters (with a default value)?

Reply:
same intension as the "0.01", we can change it to parameter later for sure.